src/aklite_client_ext.cc

```diff
@@ -55,6 +56,18 @@ GetTargetToInstallResult AkliteClientExt::GetTargetToInstall(const CheckInResult
   }
 
   const auto current = GetCurrent();
+  // It may occur that the TUF targets list only has versions lower than the current one.
+  // The `accept_lower_ver` parameter controls what to do in such situation: Should a version lower than the current
+  //  one be accepted as a valid selected target for installation or not
+  if (!accept_lower_ver && version == -1 && target_name.empty() && candidate_target.Version() < current.Version()) {
+    if (!invoke_post_cb_at_checkin_) {
+      client_->notifyTufUpdateFinished(err);
+    }
+    LOG_INFO << "Preventing selection of a downgrade target. Current: " << current.Version()
```

Comment:
Maybe `Rejecting latest target in TUF metadata to prevent downgrade; current: <>, candidate: <>`?